sgl-kernel/csrc/torch_extension.cc

```diff
@@ -22,181 +22,67 @@ TORCH_LIBRARY_EXPAND(sgl_kernel, m) {
   /*
    * From csrc/allreduce
    */
-  m.def(
```

Comment:
revert this first in https://github.com/sgl-project/sglang/pull/4871 to unblock the new release